README.md

```diff
@@ -88,13 +88,13 @@ That's it. You should be able to see more media types in your slideshows and gal
 
 Using Redis for files locking improves performance **by a factor of 10** when loading an album.
 
-Read about it in the [ownCloud](https://doc.owncloud.org/server/9/admin_manual/configuration_files/files_locking_transactional.html) Administration Manual
+Read about it in the [ownCloud](https://doc.owncloud.org/server/9.1/admin_manual/configuration_files/files_locking_transactional.html) Administration Manual
```

Comment:
You can set this to 10.0 directly as the master gallery version will be used for 10.0.